core/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsTableDescriptor.java

```diff
@@ -7,35 +7,18 @@
 import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
 import org.mobilitydata.gtfsvalidator.parsing.CsvHeader;
 
-public abstract class GtfsTableDescriptor<T extends GtfsEntity> {
+public abstract class GtfsTableDescriptor<T extends GtfsEntity> extends GtfsFileDescriptor<T> {
```

Comment:
[suggestion]: rename this class to `GtfsCsvFileDescriptor`.

Reply:
To be logical all TableDescriptor (e.g. CurrentcyAmountTableDescriptor) should also be renamed.

Reply:
This is becoming a massive change. To do things properly I would need to change any reference from Table to File. (e.g. TableLoader becomes FileLoader).
I am thinking of not doing this and having a dedicated PR for that change.